config/docker/postgresql-development/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3'
+
+services:
+  postgresql:
+    image: postgres
```

Comment:
Looks good, (I haven't checked how much works and how much doesn't).
Is this a proper image reference or do you have to be more specific?
You then have to set the proper ENV variables so that Rails knows where to find the PostgreSQL container  in the `database.yml` file. (you set the port number here so for example so you can just give rails that port number etc.)

Reply:
Also I would start by just making sure that the PostgreSQL container works.
- Start the postgresql container with docker and see if you have any errors starting.
- If it starts connect via `docker exec -it NAME_OF_CONTAINER bash` and try to connect to the postgresql instance from there.
- If you know that all works you can connect rails to the postgresql container and rails should then be able to create the database, migrate and see it (famous last words)
- To be even more thorough you can then connect to the postgresql container again and connect to the newly created database and see if the tables that rails just created are there and are functioning properly (they really should be there but I think this helps understanding :grin: )